integrations/github/src/haystack_integrations/tools/github/file_editor_tool.py

```diff
@@ -62,9 +62,6 @@ def __init__(
                 "documents": {"handler": custom_handler}
             }
         """
-        self.name = name
-        self.description = description
-        self.parameters = parameters
```

Comment:
I removed the assignment of these attributes because they are automatically assigned during `ComponentTool` initialization.

Initializing them here as `Optional` created type issues with the `Tool` base class, where they are mandatory.  `ComponentTool` also takes care of this.

Class hierarchy: GitHub Tools inherit from `ComponentTool`, which inherits from `Tool`